src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -1773,4 +1773,77 @@ get_sysfs_path(const std::string& subdev, const std::string& entry)
   return path_buf;
 }
 
+#ifdef XRT_ENABLE_AIE
```

Comment:
These functions are not required in pci/linux . These are required only in sw_emu & edge flows.